Лабораторная работа 1/task_2.py

```diff
@@ -0,0 +1,4 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+i = (100 * 50 * 25 * 4) / 1024 ** 2 # Объем памяти, занимаемый одной книгой в Мб
+count = int(1.44//i)
```

Comment:
Верное решение!
Но внесу пару корректировок:
В вашем коде есть "магические" числа.
Задания лучше решать как задачи по физике или математике. Для каждого числа необходимо создавать свою переменную.
Например:
```suggestion
disk_mb = 1.44
pages = 100
lines_per_page = 50
```
И так далее.
Таким образом, ваш код будет более понятен человеку, который не знает суть Вашего задания. Код станет более читаемым.